src/components/ExploreCard/ExploreCard.jsx

```diff
@@ -6,22 +6,54 @@ import {
   CardMedia,
   Grid,
   IconButton,
+  ListItemIcon,
+  ListItemText,
+  Menu,
+  MenuItem,
   Typography,
 } from "@mui/material";
 import { Box } from "@mui/system";
-import React from "react";
+import React, { useState } from "react";
 import { Link } from "react-router-dom";
-import DeleteIcon from "@mui/icons-material/Delete";
+import WatchLaterIcon from "@mui/icons-material/WatchLater";
+import MoreVertIcon from "@mui/icons-material/MoreVert";
+import PlaylistAddIcon from "@mui/icons-material/PlaylistAdd";
 
 export const ExploreCard = ({
-  title,
-  thumbnail,
-  creator,
-  creatorLogo,
-  slug,
-  location,
-  handleIconAction,
+  video,
+  handleModalOpen,
+  // handlePlaylist,
+  // playlist
```

Comment:
remove commented out code, if not required

Reply:
Done